<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statProg</class>
 <widget class="QDialog" name="statProg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_type_prog_stat">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>611</width>
     <height>361</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_type_fection_prog">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>571</width>
     <height>551</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="gif1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>441</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/images/output-onlinegiftools.gif);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
